klasses/bootcamp_admissions_client_test.py

```diff
@@ -0,0 +1,159 @@
+"""
+Tests for the bootcamp_admission_client module
+"""
+from unittest.mock import Mock
+
+import pytest
+from django.conf import settings
+from rest_framework import status
+
+from klasses.bootcamp_admissions_client import BootcampAdmissionClient
+
+# pylint: disable=missing-docstring,redefined-outer-name,unused-argument
+
+pytestmark = pytest.mark.django_db
+
+JSON_RESP_OBJ = {
+    "user": "foo@example.com",
+    "bootcamps": [
+        {
+            "bootcamp_id": 1,
+            "bootcamp_title": "Entrepreneurship",
+            "klasses": [
+                {
+                    "klass_id": 13,
+                    "klass_name": "Class 2 (Student)",
+                    "status": "no_show",
+                    "is_user_eligible_to_pay": False
+                },
+                {
+                    "klass_id": 12,
+                    "klass_name": "Class 1",
+                    "status": "coming",
+                    "is_user_eligible_to_pay": False
+                }
+            ]
+        },
+        {
+            "bootcamp_id": 6,
+            "bootcamp_title": "Master of Law",
+            "klasses": [
+                {
+                    "klass_id": 16,
+                    "klass_name": "Class 1",
+                    "status": "scholarship_not_awarded",
+                    "is_user_eligible_to_pay": True
+                }
+            ]
+        }
+    ]
+}
+
+
+def create_mocked_get_response(status_code, json_obj):
+    """
+    Helper function to create a mocked response for the requests.get function
+    """
+    mocked_response = Mock()
+    mocked_response.status_code = status_code
+    mocked_response.json.return_value = json_obj
+    return mocked_response
```

Comment:
sorry but why is this better than having a parameterized function that returns the object I need? What If I need another content in the JSON response? I do not think this is necessary